test/msd_test.py

```diff
@@ -382,14 +382,15 @@ def test_mass_storage(workspace, parent_test):
         with MbedBoard.chooseBoard(board_id=board.get_unique_id(), init_board=False) as mbed_board:
             memory_map = mbed_board.target.getMemoryMap()
         flash_regions = [region for region in memory_map if region.type == 'flash']
-        number_flash_regions = len(flash_regions)
 
         max_address = intel_hex.maxaddr()
         # Create an object. We'll add the addresses of unused even blocks to it first, then unused odd blocks for each region
         ih = intelhex.IntelHex()
         # Add the content from test bin first
         expected_bin_contents = bin_file_contents
         for region_index, the_region in enumerate(flash_regions):
+            if the_region.isBootMemory == False:
```

Comment:
Very minor note: use `not the_region.isBootMemory` since it reads better.

(Fyi, you don't need to change the minor notes right now, they are just suggestions for more readable code.)

Reply:
Minor thought: I usually prefer positive logic. In my mind, something == false is more readable than !something. It's okay either way.

Reply:
Understandable :)